ietf/utils/management/commands/coverage_changes.py

```diff
@@ -34,7 +34,7 @@ class Command(BaseCommand):
             "        $ manage.py {name} --absolute --sections=url | grep False\n"
             "\n".format(**locals())
         )
-    args = "[[master_json] latest_json]"
+    args = "[[main_json] latest_json]"
```

Comment:
Where is main_json (master_json) otherwise defined? Does it belong to coverage instead of this code?

Reply:
This is just the help text, I believe.